pyquil/api/qvm.py

```diff
@@ -330,3 +354,12 @@ def _add_rng_seed_to_payload(self, payload):
         """
         if self.random_seed is not None:
             payload['rng-seed'] = self.random_seed
+
+    def _add_kraus_noise_to_program(self, program):
+        """
+        Couple program to a Kraus noise model if available.
```

Comment:
Can you be more descriptive/precise? By couple do you mean apply noise to the program according to the Kraus noise model?